lib/rubocop/formatter/simple_text_formatter.rb

```diff
@@ -13,6 +13,7 @@ class SimpleTextFormatter < BaseFormatter
       include PathUtil
 
       COLOR_FOR_SEVERITY = {
+        info:       :white,
```

Comment:
This is going to break the HTML formatter because it'll make `info` severity items white as well. I think `:gray` is preferable here.

Reply:
changed.